api/python/provisioner/commands/__init__.py

```diff
@@ -678,6 +679,14 @@ def run(self, targets):  # noqa: C901 FIXME
                 update_exc, rollback_error=rollback_error
             ) from update_exc
 
+        finally:
+            if minion_conf_changes:
+                logger.debug("Restarting salt minions ")
```

Comment:
please use INFO level here

Reply:
we shouldn't ever fail here actually, I want to change my mind : let's move that  ([here](https://github.com/Seagate/cortx-prvsnr/blob/52ef5b9528246b1d4308ea23da65b90e27de6abd/api/python/provisioner/commands/__init__.py#L598)) and use try-except block
```
try:
....
except Exception as exc:
           raise SWStackUpdateError(exc) from exc
```